target_chains/solana/program_simulator/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "program-simulator"
```

Comment:
I segregated this one in case we wanna use it for another program

Reply:
thank you! would be great to avoid copy/pasting it